exercises/bowling/bowling_test.go

```diff
@@ -39,8 +47,8 @@ func TestScore(t *testing.T) {
 				}
 			}
 		} else if err == nil {
-			t.Fatalf("%s : Score() after Previous Rolls: %#v expected error \"%s\", got score %d",
-				tc.description, tc.previousRolls, tc.errorString, score)
+			t.Fatalf("FAIL: %s : Score() after Previous Rolls: %#v expected an error, got score %d\n\tExplaination: %s",
```

Comment:
spelling: Explaination -> Explanation

It does mess me up sometimes that when you explain something, you give an explanation... the i disappeared